next.config.js

```diff
@@ -1,6 +1,16 @@
 const { i18n } = require('./next-i18next.config')
 
 module.exports = {
+  async redirects() {
```

Comment:
can you add a comment similar to `The Claim Tracker application must live at /claimstatus because IDM Webgate expects the "context root" to be /claimstatus, and all other pages must be sub-paths.` here or elsewhere?

Reply:
```suggestion
  // The Claim Tracker application must live at /claimstatus because IDM Webgate 
  // expects the "context root" to be /claimstatus, and all other pages must be sub-paths.
  async redirects() {
```